xml/System.Xml/XmlDocumentXPathExtensions.xml

```diff
@@ -16,7 +16,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents XPath extension methods previously found on XML document. You can use them to create XPathNavigator given XML document object.</summary>
```

Comment:
> previously found on XML document

What does this mean exactly? If a user with zero context visits MS Docs, what would they think about that statement? Does it mean it was found on XML document back in .NET Framework?